### DISCUSSION TITLE
Does @supabase/ui have support for multi select?

### DISCUSSION THREAD

**Discussion by jparismorgan** [supabase/supabase#18804] [Questions]
Does @supabase/ui have support for multi select? I.e. something like https://react-select.com/home:

<img width="584" alt="Screenshot 2023-11-08 at 12 51 14 AM" src="https://github.com/supabase/supabase/assets/1396242/cd2b775b-8c9b-4eb7-9286-bb64fe4777e0">

I've tried with `Listbox` and `Select`, but didn't get either working. Thank you!

**Comment by GaryAustin1** [comment]
![image](https://github.com/supabase/supabase/assets/54564956/ea2f6f69-f80e-4bf3-bbdf-d53625bdb1c4)

Just a heads up it that is what you are talking about.

**Reply by jparismorgan** [reply]
![jack-sparrow-i-will-go-down-with-this-ship](https://github.com/supabase/supabase/assets/1396242/a4bb34fd-b3c2-410c-aa22-8aa3674bc576)

...just kidding, thank you, I will look at other options. (Though if someone did know that this was possible, that would be nice).

**Comment by MildTomato** [comment]
@jparismorgan [this gets you some of the way there](https://ui.shadcn.com/docs/components/combobox), at least the popover and search will work well.
You could then render some labels for each option selected and have them contained in a button for the combobox popover trigger. 

also related:
https://github.com/radix-ui/primitives/issues/1342
https://github.com/radix-ui/primitives/issues/2302
